_plugins/hook_list.rb

```diff
@@ -0,0 +1,5 @@
+# https://jekyllrb.com/docs/plugins/hooks/
+Jekyll::Hooks.register :site, :post_write do
+    # code to call after Jekyll:
+    system('cd openring && ./openring.sh')
```

Comment:
How about this draft:

```ruby
require 'mkmf'

OPENRING_LIST="openring/feed_list.txt"
OPENRING_IN="openring/openring_template_in.html"
OPENRING_OUT="_includes/openring_template_out.html"

def composing_openring_command(openring_path)
  openring_custom_parameters = ''

  File.foreach(OPENRING_LIST) do |line|
    openring_custom_parameters += " -s #{line.chomp}"
  end

  return "#{openring_path} #{openring_custom_parameters} < #{OPENRING_IN} > #{OPENRING_OUT}"
end

Jekyll::Hooks.register :site, :post_write do
    openring_path = find_executable('openring')
    openring_cmd = composing_openring_command(openring_path)

    system(openring_cmd)
end
```

With this code, you can delete the openring.sh file. But you need to improve the above code by:
* Add checks before accessing a file
* Check the OPENRING_IN file timestamp. If it was generated 15 min ago, there is no need to run this script. 
* Add documentation